tests/network/flat_overlay/utils.py

```
@@ -66,29 +67,51 @@ def create_ip_block(ip_address, ingress=True):
+            change that should trigger CNO reconciliation. Used to ensure the observed
+            `Progressing=False` transition reflects the new change rather than a stale
+            stable state. Must be timezone-aware to compare against the operator
+            `lastTransitionTime` ISO-8601 value.
+
+    Raises:
+        TimeoutExpiredError: If the operator does not reach the expected stable
+            conditions within TIMEOUT_10MIN.
+    """
+    network_co = ClusterOperator(name="network")
+
+    def _is_reconciled_and_healthy():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_is_reconciled_and_healthy`

(ANN202)